roles/skills/files/git-commit/SKILL.md

```
@@ -0,0 +1,253 @@
+
+```bash
+git diff
+```
+
+Stage what is appropriate before committing. Never use `git add -A` or `git add .` blindly — stage specific files to avoid accidentally including secrets or unrelated changes.
+
+## Step 3 — Craft the commit message
+
+Follow Conventional Commits format strictly:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)